app/components/school-vocabularies-collapsed.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { TrackedAsyncData } from 'ember-async-data';
+import { cached } from '@glimmer/tracking';
+
+export default class SchoolVocabulariesCollapsedComponent extends Component {
+  @cached
+  get vocabulariesData() {
+    return new TrackedAsyncData(this.args.school.vocabularies);
+  }
+
+  get vocabulariesLoading() {
+    return !this.vocabulariesData.isResolved;
+  }
+
+  get vocabularies() {
+    return this.vocabulariesData.isResolved ? this.vocabulariesData.value.slice() : [];
```

Comment:
```suggestion
    return this.vocabulariesData.isResolved ? this.vocabulariesData.value : [];
```
This `slice` shouldn't be needed here.